docs/configuration/binac.md

```diff
@@ -1,7 +1,7 @@
 # BINAC
```

Comment:
This can go away - its completely present on the nf-core/configs repository....

Reply:
removed the file